api/jsdoc_config.json

```diff
@@ -0,0 +1,22 @@
+{
+    "plugins": [],
+    "recurseDepth": 10,
+    "source": {
+        "exclude": ["./api/src/controllers/foo.js"]
```

Comment:
nitpick: this file will be removed in this PR https://github.com/Researchify/Researchify/pull/65

Reply:
Ok, will remove this line then